File: examples/automations/endpoints/update.py
from dotenv import load_dotenv
from retab import Retab

assert load_dotenv("../../../.env.production")

reclient = Retab()

automation = reclient.processors.automations.endpoints.update(
    endpoint_id="endp_rh-D1lcOuPZj1qqhL3YPc",
    name="Endpoint Automation Updated",
)

print(automation.model_dump_json(indent=2))

File: examples/node/automations/endpoints/update.js
import { config } from 'dotenv';
import { Retab } from '@retab/node';

config({ path: '../../../.env.local' });

const reclient = new Retab();

const automation = await reclient.processors.automations.endpoints.update({
  endpoint_id: "endp_jKH0WW6X1dD3wtWbnuuXQ",
  name: "Endpoint Automation Updated",
});

console.log(JSON.stringify(automation, null, 2));